source/reference/ejson/stringify.txt

```diff
@@ -0,0 +1,506 @@
+.. _mongosh-ejson-stringify:
+
+=================
+EJSON.stringify()
+=================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+The ``EJSON.stringify()`` method converts BSON values to strings. 
+
+Syntax
+------
+
+The ``EJSON.stringify()`` method takes a BSON object as input and
+optional modifiers that control the format of the output string.
+
+.. code-block:: javascript
+   :copyable: false
+
+   EJSON.stringify(BSON object, [replacer], [space], [options])
+
+Command Fields
+--------------
+
+The ``EJSON.stringify()`` method has these fields:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Field
+     - Type
+     - Necessity
+     - Description
+
+   * - ``value``
+     -  BSON object
+     -  Required
+     -  The object ``EJSON.stringify`` transforms
+
+   * - ``replacer``
+     -  array or function
+     -  Optional
+     -  Modifies output. If the value exists but it is not an array or a
+        function, ``EJSON.stringify`` returns all document fields. 
```

Comment:
[suggestion]

Consider updating to ``EJSON.stringify()`` to be more consistent.

Reply:
Done